docs/guide/cache/fastly.md

```diff
@@ -0,0 +1,481 @@
+---
+description: Configure Fastly for use with Ibexa DXP.
+---
+
+# Configure and customize Fastly
+
+You can configure Fastly by using API calls or through the Fastly Web Interface. 
+Fastly provides a [Fastly CLI](https://developer.fastly.com/reference/cli/) for configuring Fastly through its API.
+
+Ibexa Cloud is delivered with Fastly preconfigured. 
+It means that you don't have to do any changes to the Fastly configuration to make your site work. 
+The information provided here is only applicable if you want to change the default Fastly configuration on Ibexa Cloud, 
+or if you are not using Ibexa Cloud and want to configure Fastly to work with [[= product_name =]] on premise.
+
+!!! note "The Fastly Web Interface is not available for Ibexa Cloud"
+    It's recommend for Ibexa Cloud customers to use the Fastly CLI instead of using the Fastly API directly with `curl`, and so on.
+
+
+## Prepare for using Fastly locally
+
+These steps are not needed when you use Ibexa Cloud, because Fastly then comes preconfigured.
+
+### Get Fastly credentials from Ibexa Cloud installation
+
+To use Fastly CLI or Fastly API directly, you need to obtain the credentials for your site.
+To obtain the credentials, connect to your Fastly-enabled environment (for example, production or staging) through SSH and run the following command:
+
+``` bash
+declare|grep FASTLY
+FASTLY_KEY=...
+FASTLY_SERVICE_ID=...
+```
+
+These credentials are different for your production and staging environments. 
+When you configure the Fastly CLI, use the credentials for the environment that you want to change.
+
+You might also encounter a difference in environment variable names.
+[[= product_name =]] uses the `FASTLY_SERVICE_ID` and `FASTLY_KEY` variables but you may see `FASTLY_SERVICE_ID` and `FASTLY_API_TOKEN` in your configuration.
```

Comment:
I think this has lost a bit of context... I initially wrote that the env names used by Ibexa DXP and Fastly CLI are different... now it is not clear what we mean here IMO